src/libraries/System.Net.WebSockets.Client/ref/System.Net.WebSockets.Client.cs

```diff
@@ -43,6 +43,10 @@ public sealed partial class ClientWebSocketOptions
         public System.Net.Security.RemoteCertificateValidationCallback? RemoteCertificateValidationCallback { get { throw null; } set { } }
         [System.Runtime.Versioning.UnsupportedOSPlatformAttribute("browser")]
         public bool UseDefaultCredentials { get { throw null; } set { } }
+        [System.Runtime.Versioning.UnsupportedOSPlatformAttribute("browser")]
+        public System.Version Version { get { throw null; } set { } }
+        [System.Runtime.Versioning.UnsupportedOSPlatformAttribute("browser")]
+        public System.Net.Http.HttpVersionPolicy VersionPolicy { get { throw null; } set { } }
```

Comment:
Should this be named HttpVersion and HttpVersionPolicy?  Otherwise, it's not clear to me it's talking about the HTTP version as opposed to, say, the WebSocket version.

Reply:
I named it the same way as in HttpRequestMessage. I thought it was done so, because otherwise it has collision with [HttpVersion](https://github.com/dotnet/runtime/blob/main/src/libraries/System.Net.Primitives/src/System/Net/HttpVersion.cs) type. Should I rename it differently i.e. HttpRequestVersion?

Reply:
There shouldn't be any conflict with the HttpVersion type, other than potentially needing to qualify inside the implementation in places it may be ambiguous.